crates/bevy_color/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+//! Representations of colors in various color spaces.
+//!
+//! This crate provides a number of color representations, including:
+//!
+//! - [`Srgba`] (standard RGBA, with gamma correction)
+//! - [`LinearRgba`] (linear RGBA, without gamma correction)
+//! - [`Hsla`] (hue, saturation, lightness, alpha)
+//! - [`Lcha`] (lightness, chroma, hue, alpha)
+//! - [`Oklaba`] (hue, chroma, lightness, alpha)
+//!
+//! Each of these color spaces is represented as a distinct Rust type. Colors can be converted
+//! from one color space to another using the [`From`] trait.
+//!
+//! # Color Space Usage
```

Comment:
We should add a short note here about how you should use `Okalba` or at least `Hsla` for most color blending operations.

Reply:
To add to this, I'd like the following:
* Short comparisons between the color spaces with preferred defaults for different use cases, including when to use ColorRepresentation.
* Links to online color pickers for each color space (maybe, up to maintainers)
* Explanation of color space, gamut, display ranges, and other terms
* Some links to references on how to understand color in more detail

Reply:
>     * Links to online color pickers for each color space (maybe, up to maintainers)

Definitely an extension beyond the initial PR, but this sounds like an excellent case for a Bevy tool to be provided, perhaps in the `examples` folder. I think there's been quite a few examples of colour pickers made using the current Bevy UI system, and a colour picking tool in Bevy is a nice demonstration of how to use `bevy_color` _and_ an explanation of the broader concept of colours as well.

Reply:
I've expanded the explanation of color spaces, but not the other suggestions (gamut etc.). Mainly because I'm not confident that my explanation of these would be correct.

Reply:
Linking to other resources would be fine